lib/action_cable/channel/broadcasting.rb

```diff
@@ -0,0 +1,25 @@
+module ActionCable
+  module Channel
+    module Broadcasting
+
+      # Broadcast a hash directly to a named <tt>broadcasting</tt>. It'll automatically be JSON encoded.
+      def broadcast(broadcasting, message)
+        ActionCable.server.broadcast(broadcasting, message)
+      end
+
+      # Broadcast a hash a unique broadcasting for this <tt>object</tt> in this channel.
+      def broadcast_to(object, message)
+        broadcast(broadcasting_for([channel_name, object]), message)
+      end
+
+      # Generates a unique named broadcasting for the <tt>object</th>.
+      def broadcasting_for(object)
+        case
+        when object.is_a?(Array)               then object.map{ |o| broadcasting_for(o) }.join(':')
```

Comment:
Not loving the massive spacing needed for `then`. Let's go with newlines. Also, let's go with `model` as the var name over `object`, since we require `to_param` to be present (thus inferring we need active models).

Finally, space between map and the opening {